.tool-versions

```diff
@@ -1,4 +1,4 @@
 elixir 1.14.5-otp-25
 erlang 25.3.2.7
 python 3.9.16
-
+poetry 1.3.1
```

Comment:
question: is there a reason not to use the latest poetry (1.7.0 apparently)?

Reply:
Not really, beyond that I didn't test that. A minor version upgrade should be fine.